packages/react-native/src/Authenticator/Defaults/VerifyUser/VerifyUser.tsx

```diff
@@ -19,14 +20,17 @@ const COMPONENT_NAME = 'VerifyUser';
 const { getSkipText, getVerifyText, getAccountRecoveryInfoText } =
   authenticatorTextUtil;
 
-const censorContactInformation = (name: string, value: string): string => {
+const censorContactInformation = (
+  type: ContactMethod,
+  value: string
+): string => {
   let censoredVal = value;
-  if (name === 'email') {
+  if (type === 'Email') {
     const splitEmail = value.split('@');
     const censoredName = censorAllButFirstAndLast(splitEmail[0]);
 
     censoredVal = `${censoredName}@${splitEmail[1]}`;
-  } else if (name === 'phone') {
+  } else if (type === 'Phone Number') {
     censoredVal = censorPhoneNumber(value);
   }
   return censoredVal;
```

Comment:
Just to be clear, if the `type` !== 'Email' or 'Phone Number', the returned value is uncensored, correct?

Reply:
The available contact methods are only email or phone [and that is unlikely to change]  so there shouldn't be another option.